onnx/version_converter/adapters/pad_10_11.h

```diff
@@ -27,6 +27,8 @@ class Pad_10_11 final : public Adapter {
     node->removeAttribute(kpads);
     // Turn value attribute into input
     if (!node->hasAttribute(kmode) || node->s(kmode) == "constant") {
+      if (!node->hasAttribute(kvalue))
```

Comment:
nit: I think it would be simpler to change line 35 to
```cpp
data_value.emplace_back(node->hasAttribute(kvalue) ? node->f(kvalue) : 0.);
```
instead of inserting the attribute and then deleting it again in line 40.

Reply:
If apply your suggestion, `removeAttribute(kvalue)` makes error because there's no `value` attribute. So line 40 should also be changed like in below:

```
if (node->hasAttribute(kvalue)
    node->removeAttribute(kvalue);
```

Which do you prefer? inserting and deleting value attribute or checking `hasAttribute(kvalue)` twice?

Reply:
Makes sense. Let's go ahead merge this as-is, the difference is not important enough to iterate more.